packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -793,6 +793,71 @@ class AndroidCameraCameraX extends CameraPlatform {
     await cameraControl.setZoomRatio(zoom);
   }
 
+  /// Gets a list of video stabilization modes that are supported for the
+  /// selected camera.
```

Comment:
Overrides (here and below) don't need to copy/paste the documentation. Duplicate docs just introduce the possibility of them getting out of sync.